sympy/core/tests/test_relational.py

```diff
@@ -806,3 +859,21 @@ def test_Equality_rewrite_as_Add():
 def test_issue_15847():
     a = Ne(x*(x+y), x**2 + x*y)
     assert simplify(a) == False
+
+def test_polynomial_relation_simplification():
+    assert Ge(3*x*(x + 1) + 4, 3*x).simplify() in [Ge(x**2, -Rational(4,3)), Le(-x**2, Rational(4, 3))]
+    assert Le(-(3*x*(x + 1) + 4), -3*x).simplify() in [Ge(x**2, -Rational(4,3)), Le(-x**2, Rational(4, 3))]
+    assert ((x**2+3)*(x**2-1)+3*x >= 2*x**2).simplify() in [(x**4 + 3*x >= 3), (-x**4 - 3*x <= -3)]
+
+def test_multivariate_linear_function_simplification():
+    assert Ge(x + y, x - y).simplify() == Ge(y, 0)
```

Comment:
What happens with `Ge(x + I, y + I).simplify().subs(x,2-I).subs(y,1-I)`?

Reply:
There will be an exception for `Ge(x + I, y + I)` so the simplify will not be executed.

Reply:
I was wrong. It returns `True`. The comparison is OK with imaginary numbers as long as there are symbols in it. So my related comments elsewhere was based on an incorrect memory.

Especially `Ge(x + I, y + I).simplify()` results in the same result. Apparently I have not cancelled out constants, so  `Ge(x + 1, y + 1).simplify()` stays the same as well...

Reply:
I guess it should be possible to only cancel out real-valued constants though (or the real part of the constants rather).

Reply:
Wrong again... I had already added a check (`is_comparable` which I think was wrong anyway) in the code I tested, so no simplification at all was performed...

I see the point. I think the main question still is if complex numbers should be associated with relationals other than Eq and Ne at all? In this case it should be possible to simply factor out the imaginary part of constants and do not use that in the simplification.

Reply:
I note that `Ge(x - y + I, I)` will throw an exception, so clearly it is inconsistent that the equivalent `Ge(x + I, y + I)`  will not. It is not clear to me if there is any fundamental difference in the two formulations and if it is a feature or an issue of sympy that one can write it in a specific form to avoid the exception.

Reply:
@smichr My memory was a bit limited... I didn't add something removing constants and therefore not something keeping imaginary constants. Could be worthwhile though.